### PR TITLE
feat(contract): updated unlock strategy

### PR DESCRIPTION
**Pull request by FrenchyPeanut**

### CODE REVIEW

File: packages/hardhat/contracts/MyTvLock.sol [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+/// @title MyTvLock Contract
+/// @author Smart-Chain Team
+
+contract MyTvLock is Ownable, ReentrancyGuard {
+    using SafeMath for uint256;
+    using SafeERC20 for MyTvGovernanceToken;
+
+    struct TokenLock {
+        uint256 timestamp;
+        uint256 lastUnlockTimestamp;
```
Comment (0xSaitama) [inline packages/hardhat/contracts/MyTvLock.sol:20]: unused variable

File: packages/hardhat/contracts/MyTvLock.sol [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        require(lockForEnabled, "This function is disabled");
+
+        require(
+            myTvGovernanceToken.balanceOf(address(this)).sub(totalLocked) >=
+                amount,
+            "Contract cannot setup token lock"
+        );
+
+        AddressToTokenLock[from].timestamp = block.timestamp;
+        AddressToTokenLock[from].lastUnlockTimestamp = block.timestamp;
```
Comment (0xSaitama) [inline packages/hardhat/contracts/MyTvLock.sol:124]: useless variable

Comment (FrenchyPeanut) [reply packages/hardhat/contracts/MyTvLock.sol]: Updated.

File: packages/hardhat/contracts/MyTvLock.sol (hunk 1 above)
Comment (FrenchyPeanut) [reply packages/hardhat/contracts/MyTvLock.sol]: Updated.

**Review by 0xSaitama — APPROVED** [review]

**Pull request merged**